src/processing/mangekanter/mangekanter.md

```diff
@@ -0,0 +1,468 @@
+
+![](SprettendeTrekant1.gif "En hvit trekant som peker nedover, spretter rundt på en svart bakgrunn.")
+
+## Forbedre leseligheten {.protip}
+
+Noen ganger kan det være vanskelig å lese kall på funksjoner som tar
+mange argumenter. I Processing tar de fleste funksjoner bare imot noen
+få argumenter, men `triangle` og et par andre tar seks eller flere. Da
+kan det være nyttig å dele opp kallet over flere linjer. For eksempel
+kunne setningen ovenfor vært skrevet slik at hvert hjørne var på hver
+sin linje:
+
+```processing
+triangle(x, y,
+  x + 100, y,
+  x + 50, y + 100);
+```
+
+Hvis man fortsatt synes det er vanskelig å lese eller rotete, kan man
+legge til noen ekstra mellomrom for å få ting på linje. Merk at om man
+bruker automatisk formatering av koden i Processing, vil den fjerne
+mellomrom den mener er overflødig.
+
+## Utfordringer {.try}
+
++ Kan du tegne trekanten motsatt vei, sånn at den ser ut som en
+  pil som peker oppover istedenfor nedover?
+
+![](SprettendeTrekant2.gif "En hvit trekant som peker oppover, spretter rundt på en svart bakgrunn.")
+
++ Kan du tegne to trekanter istedenfor en og lage en sekskantet
+  stjerne?
+
+![](SprettendeSekskantetStjerne.gif "En hvit sekskantet stjerne spretter rundt på en svart bakgrunn.")
+
++ **Vanskelig**: Trekanten i programmet er nesten likesidet, men den
+  er litt for høy, så to av sidene er rundt 12 piksler for lange. Kan
+  du endre regnestykket `y + 100` sånn at trekanten blir likesidet? Du
+  kan bruke Pytagoras' læresetning, eller sinus-funksjonen, `sin` i
+  Processing, for å finne den riktige høyden. Funksjonen `radians` kan
+  være til hjelp for å gjøre om grader til radianer hvis du vil bruke
+  sinus-funksjonen.
+
+# Trekanter {.activity}
+
+Nå skal vi se hvordan vi kan lage trekanter hvor hvert hjørne beveger
+seg for seg selv. Da trenger vi fire variabler for hvert hjørne, to
+for posisjon og to for farten, og dette for tre hjørner som blir tolv
+variabler. Vi kunne kalt dem f.eks. `x1`, `x2`, `x3` og tilsvarende
+for `y`, `xFart` og `yFart`. Isteden skal vi bruke noe som kalles en
+*array*. Det er vanlig å bruke det engelske ordet også på norsk, men
+det oversettes noen ganger til vektor, rekke, tabell eller matrise.
+  
+## Sjekkliste {.check}
+
++ Vi begynner med å endre variablene til *arrays*:
+
+    ```processing
+    float[] x = new float[3];
+    float[] y = new float[3];
+    float[] xFart = new float[3];
+    float[] yFart = new float[3];
+    ```
+    
+    Nå har vi endret typen av variablene fra `float` til
+    `float[]`. Når vi putter firkantklammer etter en type, er det en
+    *array* som inneholder verdier av typen foran klammene. Bak
+    likhetstegnet ser vi også noe nytt `new float[3]` betyr at vi skal
+    lage en ny `float`-*array* med tre tall i.
+
++ Nå må vi endre startverdiene til disse tallene, ellers vil de bare
+  være `0` alle sammen:
+
+    ```processing
+    void setup() {
+      size(800, 600);
+      
+      x[0] = width / 2;

```

Comment:
Her får jeg feilmeldingen: `The type of the expression must be an array type but it resolved to float`